proposals/0425-int128.md

```diff
@@ -3,8 +3,7 @@
 * Proposal: [SE-0425](0425-int128.md)
 * Author: [Stephen Canon](https://github.com/stephentyrone)
 * Review Manager: [Doug Gregor](https://github.com/DougGregor)
-* Implementation: https://github.com/stephentyrone/swift-numerics/tree/int128
```

Comment:
The "int128" branch details could also be removed from the §"Proposed solution".